js/scripts.js

```diff
@@ -1,3 +1,14 @@
-var name = prompt('Enter your name');
-alert('Hello, ' + name);
-console.log('Hello, ' + name);
+// var name = prompt('Enter your name');
+// alert('Hello, ' + name);
+// console.log('Hello, ' + name);
+
+var a = prompt('Type a');
+var h = prompt('Type h');
+function getTriangleArea(a, h) {
+if (a, h) <= 0) {
```

Comment:
ten warunek nie działa => jeśli chcesz sprawdzić czy liczba jest równa 3 *lub* 5 zrobisz to tak `if (number === 3 || number === 5)`